index.js

```diff
@@ -101,8 +135,94 @@ module.exports = {
     'key-spacing': [2, {beforeColon: false, afterColon: true, mode: 'minimum'}],
     'brace-style': [2, '1tbs', {allowSingleLine: true}],
     'switch-colon-spacing': [2, {after: true, before: false}],
-    'semi-style': [2, 'last']
+    'semi-style': [2, 'last'],
+    'array-bracket-newline': [2, 'consistent'],
+    'array-element-newline': [2, 'consistent'],
+    'block-spacing': [2, 'always'],
+    'capitalized-comments': [2, 'always', {
+      ignoreConsecutiveComments: true,
+      ignoreInlineComments: true,
+    }],
+    'comma-dangle': [2, {
+      arrays: 'always-multiline',
+      objects: 'always-multiline',
+      imports: 'always-multiline',
+      exports: 'always-multiline',
+      functions: 'never',
+    }],
+    'default-case-last': [2],
+    'function-call-argument-newline': [2, 'consistent'],
+    'function-paren-newline': [2, 'consistent'],
+    'generator-star-spacing': [2, 'before'],
+    'implicit-arrow-linebreak': [2, 'beside'],
+    'keyword-spacing': [2, {before: true, after: true}],
+    'lines-around-comment': [2, {
+      beforeBlockComment: true,
+      beforeLineComment: true,
+      allowBlockStart: true,
+      allowBlockEnd: false,
+      allowClassStart: true,
+      allowClassEnd: false,
+      allowObjectStart: true,
+      allowObjectEnd: false,
+      allowArrayStart: true,
+      allowArrayEnd: false,
+    }],
+    'lines-between-class-members': [2, 'always'],
+    'multiline-comment-style': [2, 'separate-lines'],
+    'new-parens': [2, 'never'],
+    'newline-per-chained-call': [2, {ignoreChainWithDepth: 3}],
+    'no-inline-comments': [2],
+    'grouped-accessor-pairs': [2],
+    'no-tabs': [2],
+    'object-curly-newline': [2, {consistent: true, minProperties: 0}],
+    'object-shorthand': [2, 'consistent-as-needed'],
+    'one-var': [2, 'never'],
+    'operator-assignment': [2, 'always'],
+    'padded-blocks': [2, 'never'],
+    'prefer-regex-literals': [2],
+    'space-before-function-paren': [2, 'never'],
+    'spaced-comment': [2, 'always'],
+
+    // Unused rules.
+    'block-scoped-var': [0],
+    'id-denylist': [0],
+    'id-length': [0],
+    'id-match': [0],
+    'max-classes-per-file': [0],
+    'max-lines': [0],
```

Comment:
we could have a max here and disable it when needed. I would propose maybe 400?

Reply:
I guess it's a good number to give the developer a little prod at that point.